client/src/fields/job_titles.js

```diff
@@ -0,0 +1,44 @@
+const JOB_TITLES = [
+    {
+        name: 'Biomedical Engineer',
+        value: 'Biomedical Engineer'
+    },
+    {
+        name: 'Civil Engineer',
+        value: 'Civil Engineer'
+    },
+    {
+        name: 'Computer Engineer',
+        value: 'Computer Engineer',
+    },
+    {
+        name: 'Data Scientist',
+        value: 'Data Scientist',
+    },
+    {
+        name: 'Electrical Engineer',
+        value: 'Electrical Engineer'
+    },
+    {
+        name: 'Environmental Engineer',
+        value: 'Environmental Engineer'
+    },
+    {
+        name: 'Mechanical Engineer',
+        value: 'Mechanical Engineer'
+    },
+    {
+        name: 'Product Manager',
+        value: 'Product Manager'
+    },
+    {
+        name: 'Software Engineer',
+        value: 'Software Engineer'
+    },
+    {
+        name: 'UI/UX Designer',
+        value: 'UI/UX Designer'
+    }
```

Comment:
Not sure if we want to try to include an 'Other' option, although I imagine that could add quite a bit of complexity because we would probably have to follow up with asking what 'other' is and then keeping track of those fields and making them available as options as well... Up to y'all. We could also consider creating a backlog at some point of features we might want to include in the future and put this there?

Reply:
I like the idea of an other option!